flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -280,7 +280,7 @@
                                                     "'region': Restarts all tasks that could be affected by the task failure. "
                                                             + "More details can be found %s.",
                                                     link(
-                                                            "{{.Site.BaseURL}}docs/ops/state/task_failure_recovery/#restart-pipelined-region-failover-strategy",
+                                                            "{{.Site.BaseURL}}/{{.Site.LanguagePrefix}}/docs/ops/state/task_failure_recovery/#restart-pipelined-region-failover-strategy",
```

Comment:
```suggestion
                                                            "{{.Site.BaseURL}}{{.Site.LanguagePrefix}}/docs/ops/state/task_failure_recovery/#restart-pipelined-region-failover-strategy",
```
The LanguagePrefix already contains a leading slash.

Reply:
Thank you for pointing this out. I've fixed it in the latest commit and verify that it works.